packages/ember-runtime/lib/mixins/-proxy.js

```diff
@@ -83,9 +79,11 @@ export default Mixin.create({
   */
   content: null,
 
-  _initializeTag: on('init', function() {
+  init() {
+    this._super(...arguments);
+    meta(this).setProxy();
```

Comment:
Let's cache the meta here:

```
let m = meta(this);
m.setProxy();
m._tag = new ProxyTag(this)
```

It isn't a huge deal, just saves a little extra work.